test/docs/wrap_test.rb

```diff
@@ -281,4 +219,96 @@ def self.call((ctx), *, &block)
     it { Memo::Create.( { seq: [] } ).inspect(:seq).must_equal %{<Result:true [[:find_model, :update, :rehash, :notify]] >} }
     it { Memo::Create.( { seq: [], rehash_raise: true } ).inspect(:seq).must_equal %{<Result:true [[:find_model, :update, :rehash, :notify]] >} }
   end
+
+=begin
+When success: return the block's returns
+When raise:   return {true} and go "successful"
+You can also return booleans in wrap.
+=end
+  class WrapGoesIntoBooleanFromRescueTest < Minitest::Spec
+    Memo = Module.new
+
+    class Memo::Create < Trailblazer::Operation
+      class HandleUnsafeProcess
+        def self.call((ctx), *, &block)
+          yield # calls the wrapped steps
+        rescue
+          ctx[:wrap_boolean_return_value] # can be true/false/nil
+        end
+      end
+
+      step :find_model
+      step Wrap( HandleUnsafeProcess ) {
+        step :update
+        step :rehash
+      }
+      step :notify
+      fail :log_error
+
+      #~methods
+      include T.def_steps(:find_model, :update, :notify, :log_error)
+      include Rehash
+      #~methods end
+    end
+
+    it "translates true returned form a wrap to a signal with a `success` semantic" do
+      result = Memo::Create.( { seq: [], rehash_raise: true, wrap_boolean_return_value: true } )
```

Comment:
Hey @KamilMilewski , examples looks good! But I'm wondering if we should control the output behaviour from external flag. Wouldn't it be confusing later that even though the exception is getting raised, the activity is still successful (unlike circuit int example) ?

Reply:
@yogeshjain999 yep, those flags looked fishy to me too, maybe lets write separate examples for every case, that is true/false/nil ?

Reply:
Agreed @KamilMilewski. I guess we can keep it simple and return `false` from `rescue` when `rehash_raise ` is true ??

Reply:
@yogeshjain999 yep, I will update it to be this way :+1: ty for suggestions

Reply:
@yogeshjain999 done